autotx/load_tokens.py

```diff
@@ -5,17 +5,31 @@
 import requests
 
 KLEROS_TOKENS_LIST = "https://t2crtokens.eth.link/"
-COINGECKO_TOKENS_LIST = "https://tokens.coingecko.com/uniswap/all.json"
+COINGECKO_TOKENS_LISTS = [
+    "https://tokens.coingecko.com/uniswap/all.json",
+    "https://tokens.coingecko.com/ethereum/all.json",
```

Comment:
Are we handling duplicates? Also do you think it makes sense to check the the final token list into github so it can be viewed? How large is it?

Reply:
> Are we handling duplicates?

when it comes to [add the tokens](https://github.com/polywrap/AutoTx/blob/main/autotx/utils/ethereum/networks.py#L32-L37) into the network dictionary, we're adding those based on the symbol, if the symbol already exists, it will be overwritten. I added the `uniswap/all.json` first on purpose, so if there's a token repeated in below lists then it will take the last  

> Also do you think it makes sense to check the the final token list into github so it can be viewed?

this would improve _a bit_ the dev experience (because we would avoid the need of running `load_tokens`) so i think it might make sense imo

>  How large is it?

116k lines with these additions :/

Reply:
I think the only 2 downsides I see to adding it to git is (1) it skews the # of lines committed by a single contributor and (2) it makes the size of the repo a bit larger. Both of these are very minor issues, so I would rather us remove the setup step which improves new devexp, and avoids this setup step potentially failing for someone in the future. Let's check it in :)

Reply:
okkk - what i would recommend then is to not remove the setup step (since it will allow us to easily update the file with the script) but only committing it, if in any case we would like to add/remove a list we could just add the URL and run the script - wdyt?